go.mod

```diff
@@ -13,6 +13,7 @@ go 1.17
 require (
 	github.com/BurntSushi/toml v1.0.0
 	github.com/alessio/shellescape v1.4.1
+	github.com/blang/semver/v4 v4.0.0
```

Comment:
I think we can avoid it https://github.com/kubernetes-sigs/kind/pull/3614/files#r1601717270

Reply:
Done!